src/components/GenerateCVButton/index.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import { PropTypes as PT } from 'prop-types';
 import { buttonTypes } from 'Constants';
-import { Button } from 'Common';
 import container from './container';
+import StyledGenerateCVButton from './styled';
 
 const GenerateButton = ({ disabled, handleClick }) => {
   return (
-    <Button
+    <StyledGenerateCVButton
       value="Generate CV"
       disabled={disabled}
       type={buttonTypes.SUBMIT}
       handleClick={handleClick}
+      buttonClass="primary"
```

Comment:
Might be better to have ButtonType, keep anything CSS related or inferred inside the styled component. Just a suggestion though.

Could also have a ButtonTypes constant.